scout/parse/variant/genotype.py

```diff
@@ -411,21 +420,77 @@ def _parse_format_entry(variant, pos, format_entry_name):
     alt = None
     if format_entry_name in variant.FORMAT:
         try:
-            value = variant.format(format_entry_name)[pos]
-            values = list(value.split("/"))
+            values = variant.format(format_entry_name)[pos]
+
+            new_values = []
```

Comment:
Otherwise this could also work

values = re.split("/|,", values)

Reply:
Yep, nice, and I would in perl! 😁 I believe Python RE is slightly more machinery than needed here, and it still would not work on arrays directly so would have to loop it.